src/frontend/src/test-e2e/util.ts

```diff
@@ -3,56 +3,57 @@ import { command } from "webdriver";
 import * as SeleniumStandalone from "selenium-standalone";
 import { ChildProcess } from "selenium-standalone";
 
+const MOBILE_SCREEN: ScreenConfiguration = {
+  screenType: "mobile",
+  windowSize: "360,640",
+};
+
+const DESKTOP_SCREEN: ScreenConfiguration = {
+  screenType: "desktop",
+  windowSize: "1920,1080",
```

Comment:
out of curiosity, why not keep the previous screen size (1050,1400) ?

Reply:
According to my quick research, this should be the most widely used desktop resolution. Especially the width 1050 seems a bit narrow. I think it's useful to have a more "extreme" width because layout issues may be easier visible. Another popular one seems to be 1366, but I wonder if the typical user (with e.g. high resolution macbook screen) will open the site in that size.